lib/iron_worker_ng/feature/python/merge_requirements.rb

```diff
@@ -0,0 +1,35 @@
+module IronWorkerNG
+  module Feature
+    module Python
+      module MergeRequirements
+        module InstanceMethods
+
+          def merge_requirements(path)
+            feature = IronWorkerNG::Feature::Base.new(self)
+            IronWorkerNG::Fetcher.fetch_to_file(feature.rebase(path)) do |requirements|
```

Comment:
merge_pip(name, version = '') adds a pip name with version to the list of packages for installing.
First we need to parse requirements.txt to get a list of packages. The question is, are we parsing requirements.txt properly?